samcli/commands/_utils/options.py

```diff
@@ -56,6 +56,24 @@ def get_or_default_template_file_name(ctx, param, provided_value, include_build)
     return result
 
 
+def guided_deploy_stack_name(ctx, param, provided_value):
+    """
+    Provide a default value for stack name if invoked with a guided deploy.
+    :param ctx:
+    :param param:
+    :param provided_value:
+    :return:
+    """
+
+    default_stack_name = "sam-app"
```

Comment:
nit-pick: I am not sure if this is a convention we are following but at few places I have seen constants to be in all CAPS like: `DEFAULT_STACK_NAME`.